sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -77,6 +76,7 @@ def publish_cloud_events(
         topic_name: str,
         body: CloudEvent,
         *,
+        binary_mode: Optional[bool] = False,
```

Comment:
What happens if someone passes in None explicitly?

Reply:
I'm assuming this should just be the following since the docstrings don't specify None:
```suggestion
        binary_mode: bool = False,
```